src/main/java/soc/client/MessageHandler.java

```diff
@@ -1305,8 +1305,6 @@ protected void handleSITDOWN(final SOCSitDown mes)
             {
                 System.out.println("Exception caught - " + e);
                 e.printStackTrace();
-
-                return;
```

Comment:
If there's an exception here, we do need to return instead of continuing handleSITDOWN

Reply:
What's the point of the finally then?

We should either move EVERYTHING into the try/catch/finally?

Reply:
The finally makes sure ga.releaseMonitor() happens, even if it's returning from the catch.

The try/catch/finally is about adding the player to the game, but the method has other things to do after that. Better to have it stay as-is and not move other things into the finally, I think :)

Reply:
To avoid locking problems, the scope between ga.takeMonitor() / ga.releaseMonitor() should be kept as small as possible. Please don't move any code into the try/catch/finally.